src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -498,7 +533,7 @@ protected virtual void CompleteNode(ExecutionContext context, ExecutionNode node
             }
             catch (Exception ex)
             {
-                if (ProcessNodeUnhandledException(context, node, ex))
+                if (ProcessNodeUnhandledException(context, node, ex)) //TODO: change? No exception from user resolver here
```

Comment:
This is necessary.  Any time that a resolver returns a value that is invalid for the graph type, it causes an error to be thrown within Serialize.  This is an unhandled "server" exception and must be processed by the unhandled exception delegate, or depending on the configuration of the execution options, throw immediately back to the caller of ExecuteAsync with no wrapping.

Reply:
```suggestion
                if (ProcessNodeUnhandledException(context, node, ex))
```

Reply:
It seems now I understood. I was misled for the comment on the ProcessNodeUnhandledException method. I changed it.